modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc

```diff
@@ -0,0 +1,105 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {prod-short} on the {eks} cluster
+:description: Installing {prod-short} on the {eks} cluster
+:keywords: installing-{prod-short}-on-the-eks-cluster
+:page-aliases:
+
+[id="installing-{prod-short}-on-AWS-EKS.adoc"]
+= Installing {prod-short} on the {eks} cluster
+
```

Comment:
docs usually have a short introductory paragraph about the process they describe. can you add it, please?

examples: 

- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-the-virtual-kubernetes-cluster/
- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-microsoft-azure/
- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-openshift-using-cli/